imports/plugins/core/orders/client/templates/workflow/shippingInvoice.js

```diff
@@ -7,6 +7,7 @@ import { ReactiveVar } from "meteor/reactive-var";
 import { Reaction, i18next, Logger } from "/client/api";
 import { NumericInput } from "/imports/plugins/core/ui/client/components";
 import { Media, Orders, Shops } from "/lib/collections";
+import _ from "lodash";
```

Comment:
I would suggest one step further in

import npm
import meteor
import third-party
import local